apps/homescreen/test/unit/pages_test.js

```diff
@@ -314,6 +321,34 @@ suite('Pages', () => {
       };
       pages.unpinSelectedCard();
     });
+
+    suite('in case of DataStore conflict', function(done) {
```

Comment:
For Mocha handlers, lexically scoped functions are not supported (they don't get the right Mocha context attached), so even though for most things they still work without the proper context, I prefer to use `function` for all Mocha callbacks. Not sure if you want to mix this in one file, I can change it either way.